pom.xml

```diff
@@ -364,6 +370,22 @@
             </build>
         </profile>
     </profiles>
+<!--    <reporting>-->
```

Comment:
I'm not getting great reporting from the Dependency Check plugin. I'd appreciate any advice about  how to best capture its results. As it stands, Circle CI storing the top level dependency_check.html file as an artifact.

Reply:
It prints out failure in the console. I don't think we need to worry about keeping the dependency_check.html file.